source/FreeRTOS_Sockets.c

```diff
@@ -1924,6 +1924,7 @@ BaseType_t FreeRTOS_setsockopt( Socket_t xSocket,
                          * when there is an event the socket's owner might want to
                          * process. */
                         /* The type cast of the pointer expression "A" to type "B" removes const qualifier from the pointed to type. */
```

Comment:
This needs a justification.

Reply:
~~~c
/* The type cast of the pointer expression "A" to type "B"
 * removes const qualifier from the pointed to type. */
~~~
A justification could be that we're copying a memory address that points to the start of a function. There is no intention to change the value of the pointee.